### DISCUSSION TITLE
I am trying to get AI2Thor to run in Windows.  The YZ article (in Medium) was helpful, but wasn't quite enough to get me there.

### DISCUSSION THREAD

**Discussion by jimjhb** [allenai/ai2thor#1186] [General]
I pulled the code (did not use pip install).  The Unity sub-directory seemed to load into Unity, which is great, but the call to Controller (as presented by YZ) just hangs.  I think it is trying to access the Unity server, but I am not sure.  The article is helpful, but it is out of date, and the suggested fixes do not represent present day code.

**Comment by jimjhb** [comment]
We managed to get it working.  Our AI2Thor now runs on Windows.

**Reply by aleSuglia** [reply]
@jimjhb could you please share how you managed to do that? Thanks!

**Reply by ML72** [reply]
@jimjhb @aleSuglia is there an update on this? I am actively trying to run AI2Thor on windows as well, but am running into display issues (even in headless mode)
